logging/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "nodejs-docs-samples-logging",
+  "description": "Node.js samples for Google Cloud Logging.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache Version 2.0",
+  "scripts": {
+    "list": "node list.js",
+    "write": "node write.js",
+    "export": "node export.js"
+  },
+  "dependencies": {
+    "gcloud": "^0.27.0"
+  }
```

Comment:
Sure these samples can be run locally and have their own dependencies, but nothing about that cares about the `engines` field.